### DISCUSSION TITLE
Transfer or archive `mkautodoc`.

### DISCUSSION THREAD

**Discussion by lovelydinosaur** [mkdocs/mkdocs#3832] [General]
I'm not currently maintaining the `mkautodoc` plugin, and considering changing permissions on it into `public archive`. I'm also open to transferring ownership instead if there's any demand. (I assume it's probably not widely used these days, tho 🤷🏼‍♂️.)

**Comment by pawamoy** [comment]
Happy to help migrating your projects using mkdautodoc over to mkdocstrings, if you want/need :slightly_smiling_face:
